goto.py

```diff
@@ -86,6 +91,28 @@ def _parse_instructions(code):
         extended_arg_offset = None
         yield (dis.opname[opcode], oparg, offset)
 
+def _get_instruction_size(opname, oparg=0):
+    size = 1
+
+    extended_arg = oparg >> _BYTECODE.argument_bits
+    if extended_arg != 0:
+        size += _get_instruction_size('EXTENDED_ARG', extended_arg)
+        oparg &= (1 << _BYTECODE.argument_bits) - 1
+
+    opcode = dis.opmap[opname]
+    if opcode >= _BYTECODE.have_argument:
+        size += _BYTECODE.argument.size
+
+    return size
+
+def _get_instructions_size(ops):
```

Comment:
It seems you did a mistake during merge. This function, and the one above are now defined twice, see below.

Reply:
Yeah, merge failure. I see you added flake8 which catches this, I'll fix it (in the other PR).